addons/mail/wizard/mail_compose_message.py

```diff
@@ -126,9 +126,11 @@ def default_get(self, fields):
         'mail.activity.type', 'Mail Activity Type',
         index=True, ondelete='set null')
     # destination
-    reply_to = fields.Char('Reply-To', help='Reply email address. Setting the reply_to bypasses the automatic thread creation.')
-    no_auto_thread = fields.Boolean(
-        'No threading for answers',
+    reply_to = fields.Char('Reply-To-Email', help='Reply email address. Setting the reply_to bypasses the automatic thread creation.')
+    reply_to_target = fields.Selection([
+        ('original_discussion', 'Original Discussion'),
+        ('another_email_address', 'Another Email Address')],
+        string='Reply-To', default='original_discussion',
         help='Answers do not go in the original document discussion thread. This has an impact on the generated message-id.')
```

Comment:
Update to help to match the selections : 
original_discussion = > Explain what it does exactly and what is the impact
another_email_address => Explain what it does exactly and what is the impact

Reply:
> Update the help to match the selections :
> original_discussion = > Explain what it does exactly and what is the impact
> another_email_address => Explain what it does exactly and what is the impact

Done !!